overview.dd

```diff
@@ -419,12 +414,10 @@ $(SECTION3 $(LNAME2 productivity, Productivity),
 
     $(SECTION4 Declaration vs Definition,
 
-    $(P C++ usually requires that functions and classes be declared twice - the declaration
-    that goes in the .h header file, and the definition that goes in the .c source
-    file. This is an error prone and tedious process. Obviously, the programmer
-    should only need to write it once, and the compiler should then extract the
-    declaration information and make it available for symbolic importing. This is
-    exactly how D works.
+    $(P Functions and classes are defined once. There is no need
+    for declarations when they are forward referenced.
```

Comment:
`no need for declarations` => `no need for forward declarations`

An example speaks more than words though, I'd add this below:

```
For example:

-----
class C : D { }  // D is already visible from here
class D { }  // does not need to be forward-declared
-----
```